samples/container/kafka-function/README.md

```diff
@@ -0,0 +1,77 @@
+# Custom Container with Kafka extension
+
+You can use Kafka extension with custom container. If you don't know about the custom container, you can refer to [Create a function on Linux using a custom container](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-linux-custom-image?tabs=bash%2Cportal&pivots=programming-language-csharp).
+
+This sample is for java, however, you can pick any languages. You can follow the step of the [Create a function on Linux using a custom container](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-linux-custom-image?tabs=bash%2Cportal&pivots=programming-language-csharp) with additional steps only for the Kafka extensions.
+
+## kafka extension additional steps
+
+1. Install the kafka library (Java and Python only)
+2. Install the kafka extension
+3. Set the LD_LIBRARY_PATH
+
+For the 2. and 3. the step is included on the `Dockerfile`.
+
+# Quick Start
+
+## Install the kafka library
+
+Before building your Dockerfile, please install kafka library for Java and Python. This step will be removed in the near future.
+If you want to try other languages, you can refer the other language samples to know how to write a function and how to install kafka extension and library.
+
+Follow the step of the [Build and Install](../../../binding-library/java/README.md#build-and-install). 
+
+You need to change the install step. This command install the java kafaka library to the `src/repo` as a local repository. The `pom.xml` file configure the local repository on the directory.
+
+```bash
+$ mvn install:install-file -Dfile=target/azure-functions-java-library-kafka-1.0.0.jar -DgroupId=com.microsoft.azure.functions -DartifactId=azure-functions-java-library-kafka -Dversion=1.0.0 -Dpackaging=jar -DlocalRepositoryPath=../../samples/container/kafka-function/src/repo/
+```
+
+## Build, Test, Publish the Docker container
+
+Refer to the [Dockerfile](./Dockerfile). The `Dockerfile` execute these steps. 
+
+* mvn clean package
+* Install Kafka Extension
+* LD_LIBRARY_PATH as an enviornment variables
+
+### Build
+
+```bash
+$ docker build --tag {YOUR_DOCKERHUB_NAME}/azurefunctionsimage:v1.0.0 .
```

Comment:
Can we update the current Dockerfile we have for java to align our latest image.

Reply:
Hi @amamounelsayed , Currently, I'm using `mcr.microsoft.com/azure-functions/java:3.0-java8-appservice` image. Which is the latest image?

Reply:
https://github.com/Azure/azure-functions-kafka-extension/blob/dev/samples/java/Dockerfile, I will send you the latest Dockerfile we have in functions.

Reply:
It is very good catch! I didn't notice it!. I'll keep it as is on this PR. however, I create an issue try not to forget this fact. 
https://github.com/Azure/azure-functions-kafka-extension/issues/141